modules/candidate_parameters/jsx/ParticipantStatus.js

```diff
@@ -226,7 +226,11 @@ var ParticipantStatus = React.createClass(
       var self = this;
       var formData = new FormData();
       for (var key in myFormData) {
-        if (myFormData[key] !== "") {
+        if (myFormData.hasOwnProperty(key) &&
+          myFormData[key] !== "" &&
+          myFormData[key] !== null &&
+          myFormData[key] !== undefined
```

Comment:
It may be better to simply use `!=` here due to javascript coercion. http://webreflection.blogspot.com/2010/10/javascript-coercion-demystified.html

I will look into this more though to confirm.

Reply:
After reading through it appears that `myFormData[key] != null` would in fact be a more concise check.

Reply:
I don't think myFormData will ever hold an object though, it would only hold values from the form, no?

Reply:
> After reading through it appears that myFormData[key] != null would in fact be a more concise check.
i would rather be explicit. i generally do not like using loose comparisons unless i have a good reason for doing so.

> I don't think myFormData will ever hold an object though, it would only hold values from the form, no?
i don't think i know what you're talking about here

Reply:
I don't think I know what I'm talking about either.